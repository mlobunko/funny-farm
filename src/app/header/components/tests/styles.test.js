import React from 'react';
import { shallow } from 'enzyme';
import 'enzyme-to-json';

import {
  StyledH1,
  StyledHeaderComponent,
  StyledLeft,
  StyledMusicIcon,
  StyledRight
} from '../styles';

describe('StyledH1', () => {
  it('renders correctly', () => {
    const wrappper = shallow(<StyledH1 />);
    expect(wrappper).toMatchSnapshot();
  });
});

describe('StyledHeaderComponent', () => {
  it('renders correctly', () => {
    const wrappper = shallow(<StyledHeaderComponent />);
    expect(wrappper).toMatchSnapshot();
  });
});

describe('StyledLeft', () => {
  it('renders correctly', () => {
    const wrappper = shallow(<StyledLeft />);
    expect(wrappper).toMatchSnapshot();
  });
});

describe('StyledMusicIcon', () => {
  it('renders soundOn', () => {
    const isMusicOn = true;
    const wrappper = shallow(<StyledMusicIcon isMusicOn={isMusicOn} />);
    expect(wrappper).toMatchSnapshot();
  });

  it('renders soundOff', () => {
    const isMusicOn = false;
    const wrappper = shallow(<StyledMusicIcon isMusicOn={isMusicOn} />);
    expect(wrappper).toMatchSnapshot();
  });
});

describe('StyledRight', () => {
  it('renders correctly', () => {
    const wrappper = shallow(<StyledRight />);
    expect(wrappper).toMatchSnapshot();
  });
});
