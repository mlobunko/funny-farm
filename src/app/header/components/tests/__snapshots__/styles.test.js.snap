// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledH1 renders correctly 1`] = `
<h1
  className="sc-bdVaJa FZjlp"
/>
`;

exports[`StyledHeaderComponent renders correctly 1`] = `
<div
  className="sc-bwzfXH fHzpjp"
/>
`;

exports[`StyledLeft renders correctly 1`] = `
<div
  className="sc-htpNat kNeozb"
/>
`;

exports[`StyledMusicIcon renders soundOff 1`] = `
<div
  aria-label="toggle play music"
  className="sc-bxivhb dXjchO"
/>
`;

exports[`StyledMusicIcon renders soundOn 1`] = `
<div
  aria-label="toggle play music"
  className="sc-bxivhb bLTPXg"
/>
`;

exports[`StyledRight renders correctly 1`] = `
<div
  className="sc-ifAKCX eYtzkY"
/>
`;
